src/assets/data/SearchCarouselData.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "name": "Get that perfect fit",
+    "image": "https://s3-media0.fl.yelpcdn.com/educatorphoto/GYdiDg2Q9tLladq7jQ6FVw/o.jpg",
+    "buttonText": "Sewing Alterations",
+    "category": ""
```

Comment:
Why is this empty?

Reply:
Before i couldn't find the list having these category codes.  I had it noted in my changes list and i have completed it now.